From 94844e6d143149cf27495ab3a402bb58b36e549c Mon Sep 17 00:00:00 2001
Message-Id: <94844e6d143149cf27495ab3a402bb58b36e549c.1528150266.git.marcel.ziswiler@toradex.com>
From: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Date: Tue, 5 Jun 2018 00:08:04 +0200
Subject: [PATCH 1/2] apalis_t30: add configuration for tezi

Modify configuration for Tezi:
- Use distroboot by default
- Boot from external SD cards by default
- Disable USB PID from config block (otherwise recovery mode download
  in U-Boot stage fails)

Signed-off-by: Marcel Ziswiler <marcel.ziswiler@toradex.com>
---
 include/configs/apalis_t30.h | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/include/configs/apalis_t30.h b/include/configs/apalis_t30.h
index 2896c522e7..8cacbd4dee 100644
--- a/include/configs/apalis_t30.h
+++ b/include/configs/apalis_t30.h
@@ -44,12 +44,14 @@
 #define CONFIG_GENERIC_MMC
 #define CONFIG_SUPPORT_EMMC_BOOT	/* eMMC specific */
 
+#define CONFIG_ENV_IS_NOWHERE
+#ifdef CONFIG_ENV_IS_IN_MMC
 /* Environment in eMMC, before config block at the end of 1st "boot sector" */
-#define CONFIG_ENV_IS_IN_MMC
 #define CONFIG_ENV_OFFSET		(-CONFIG_ENV_SIZE + \
 					 CONFIG_TDX_CFG_BLOCK_OFFSET)
 #define CONFIG_SYS_MMC_ENV_DEV		0
 #define CONFIG_SYS_MMC_ENV_PART		1
+#endif
 
 /* USB host support */
 #define CONFIG_USB_EHCI
@@ -82,8 +84,7 @@
 #define CONFIG_SERVERIP		192.168.10.1
 
 #define CONFIG_BOOTCOMMAND \
-	"run emmcboot; setenv fdtfile ${soc}-apalis-${fdt_board}.dtb && " \
-		"run distro_bootcmd"
+	"run distro_bootcmd"
 
 #define DFU_ALT_EMMC_INFO	"apalis_t30.img raw 0x0 0x500 mmcpart 1; " \
 				"boot part 0 1 mmcpart 0; " \
-- 
2.14.4

